src/win/util.c

```diff
@@ -512,19 +512,22 @@ int uv_uptime(double* uptime) {
 
 
 unsigned int uv_available_parallelism(void) {
-  SYSTEM_INFO info;
-  unsigned rc;
+  DWORD_PTR procmask;
+  int count;
+  int i;
 
   /* TODO(bnoordhuis) Use GetLogicalProcessorInformationEx() to support systems
    * with > 64 CPUs? See https://github.com/libuv/libuv/pull/3458
    */
-  GetSystemInfo(&info);
+  count = 0;
+  if (GetProcessAffinityMask(GetCurrentProcess(), &procmask, NULL))
```

Comment:
Does the system mask need to be non-NULL?  I got a ReadOnlyMemoryError in Julia when I tried to pass NULL to that argument.

Reply:
Maybe wine is a little more permissive than actual windows. Fix pushed.